deltacat/utils/profiling.py

```diff
@@ -0,0 +1,39 @@
+import ray
```

Comment:
Since we don't have anything related to profiling in this module yet, maybe we should just rename it to something more specific like `resources.py`?